src/components/common/NameRequestInfo.vue

```diff
@@ -0,0 +1,161 @@
+<template>
+  <div id="name-request-summary">
+    <v-row >
+      <v-col id="name-request-info" cols="4">
+        <ul>
+          <li> <label>
+          <strong>Name Request</strong>
+        </label></li>
+          <li>Entity Type: {{ entityTypeDescription() }}</li>
+          <li> Request Type: {{ requestType() }}</li>
+          <li>Expiry Date: {{ formattedExpirationDate() }}</li>
+          <li>
+            <v-icon v-if="nameRequestDetails.status === NameRequestStates.APPROVED" color="green"
+            class="nr-status-icon">mdi-check</v-icon>
+            Status: {{ formatStatusText(nameRequestDetails.status) }}
+          </li>
+          <li id="condition-consent">
+            <v-icon v-if="conditionConsent() === 'Received' || conditionConsent() === 'Not Required'"
```

Comment:
Please use a constant (key) here instead of a string (so that someone can't change this code and conditionConsent() separately).
Example: https://github.com/bcgov/business-filings-ui/blob/f8d33f077fa8e0626fc5969725cfd71a5ff609a1/src/mixins/date-mixin.ts#L8

Reply:
Done